src/VisualStudio/Core/Def/Implementation/ProjectSystem/CPS/IWorkspaceProjectContextFactory.cs

```diff
@@ -17,7 +15,11 @@ internal interface IWorkspaceProjectContextFactory
     {
         /// <inheritdoc cref="CreateProjectContextAsync"/>
         [Obsolete("Use CreateProjectContextAsync instead")]
-        IWorkspaceProjectContext CreateProjectContext(string languageName, string projectUniqueName, string projectFilePath, Guid projectGuid, object hierarchy, string binOutputPath);
+        IWorkspaceProjectContext CreateProjectContext(string languageName, string projectUniqueName, string projectFilePath, Guid projectGuid, object? hierarchy, string? binOutputPath);
+
+        /// <inheritdoc cref="CreateProjectContextAsync"/>
+        [Obsolete("Use CreateProjectContextAsync instead")]
+        IWorkspaceProjectContext CreateProjectContext(string languageName, string projectUniqueName, string projectFilePath, Guid projectGuid, object? hierarchy, string? binOutputPath, string? assemblyName);
```

Comment:
😕

Reply:
So this is "fun": we obsoleted those methods, the project system tried moving to the new methods; that broke, so they rolled back, and we haven't fixed it. @CyrusNajmabadi what's the next step for resolving that?

Reply:
There was an RPS regression when we tried using the new free-threaded API. The logs for that regression have no expired, so to understand that regression will require another PS insertion.